README.md

```diff
@@ -24,15 +24,150 @@ ACM-ICPC Algorithms is a collection of important algorithms and data structures
 * [Breadth First Search](/BFS)
 * [Binary Search Tree](/BST)
 * [Backtracking](/BackTracking)
+   * [Knights Tour](/BackTracking/KnightsTour)
+   * [NQueens](/BackTracking/NQueens)
+   * [Rat In A Maze](/BackTracking/RatInAMaze)
+   * [Sudoku Algorithm](/BackTracking/SudokuAlgorithm)
 * [Depth First Search](/DFS)
 * [Bit Manipulation](/BitManipulation)
+   * [Checking Power of 2](/BitManipulation/Checking_power_of_2)
+   * [Count Ones](/BitManipulation/count_ones)
+   * [Divide Integers](/BitManipulation/divide_integers)
+   * [Even Odd](/BitManipulation/even_odd)
+   * [Print Subsets](/BitManipulation/print_subsets)
+   * [Reverse Bits](/BitManipulation/reverse_bits)
+   * [Single Number](/BitManipulation/single_number)
+   * [Swap Bits](/BitManipulation/swap_bits)
 * [Data Structures](/Data%20Structures)
+   * [Disjoint Set](/Data%20Structures/disjointset)
+   * [Doubly Linked List](/Data%20Structures/DoublyLinkedList)
+   * [Fenwick Tree](/Data%20Structures/Fenwick_tree)
+   * [LCA](/Data%20Structures/LCA)
+   * [Linked List](/Data%20Structures/Linked%20List)
+   * [Queue](/Data%20Structures/Queue)
+   * [Queue From Stack Or Stack From Queue](/Data%20Structures/QueueFromStack_StackFromQueue)
+   * [Red Black Tree](/Data%20Structures/Red%20Black%20Tree)
+   * [Singly Linked List](/Data%20Structures/SinglyLinkedList)
+   * [Stack](/Data%20Structures/Stack)
+   * [Segment Tree](/Data%20Structures/Segment%20Tree)
+   * [Trie](/Data%20Structures/Trie)
 * [Dynamic Programming](/DP)
-* [Graph Algorithms](/Graph) 
-* [Greedy Algorithms](/Greedy/Kruskal’sMinimumSpanningTree)
-* [Hashing Algorithms](/Hashing) 
+   * [Coin Change](/DP/Coin%20Change%20Problem)
+   * [Collect Maximum Points](/DP/Collect_Max_Points)
+   * [Edit Distance](/DP/EditDistance)
+   * [Fibonacci Series](/DP/Fibonacci)
+   * [Floyd Warshall Algorithm](/DP/Floyd%20Warshall%20Algorithm)
+   * [Game Of Sum](/DP/game_of_sum)
+   * [Knapsack](/DP/Knapsack)
+   * [Longest Common Independent Subsequence](/DP/LCIS)
+   * [Longest Common Subsequence](/DP/LongestCommonSubsequence)
+   * [Longest Increasing Subsequence](/DP/LongestIncreasingSubsequence)
+   * [Longest Repeated Subsequence](/DP/Longest%20Repeated%20Subsequence)
+   * [Matrix Chain Multiplication](/DP/MatrixChain_multiplication)
+   * [Max Sum Increasing Subsequence](/DP/Maximum%20Sum%20Increasing%20Subsequence)
+   * [Minimum Path Sum](/DP/MinimumPathSum)
+   * [Number Of Islands](/DP/NumberOfIslands)
+   * [Print Neatly](/DP/PrintNeatly)
+   * [Shortest Uncommon Subsequence](/DP/ShortestUncommonSubsequence)
+   * [Subset Sum](/DP/subset%20sum%20problem)
```

Comment:
Change to `/DP/subset_sum%20problem`.

Reply:
@matthewsamuel95 I guess I was 13 minutes too slow haha (Simply fix regardless)

Reply:
👏

Reply:
:)